osu.Game.Rulesets.Osu/Difficulty/Skills/Flashlight.cs

```diff
@@ -28,6 +28,7 @@ public Flashlight(Mod[] mods)
         private double skillMultiplier => 0.05;
         private double strainDecayBase => 0.15;
         protected override double DecayWeight => 1.0;
+        protected override int ReducedSectionCount => -1;
```

Comment:
On the code side -1 is really weird as well as 0 should work the same and actually makes semantic sense.